util/attr/attr.js

```diff
@@ -38,7 +38,18 @@ steal("can/util/can.js", function (can) {
 			 * `element.getAttribute` and `element.setAttribute`.
 			 */
 			map: {
-				"class": "className",
+				"class": function(el, val) {
+					val = val || '';
+
+					if(el instanceof SVGElement) {
```

Comment:
Thinking of IE8, this won't work. I'll update to `if(el.namespaceURI === 'http://www.w3.org/2000/svg')` like [modernizr](https://gist.github.com/mstalfoort/1293822#file-gistfile1-html-L45).